packages/extension-points/index.js

```diff
@@ -16,21 +16,30 @@ Promise.all([
       operatingDirectory,
     )}.admin_ui.components.yml`,
   ),
+  readFilePM(
```

Comment:
What's that?

Reply:
`const [readFilePM] = [readFile].map(promisify);`